opentelemetry-api/src/opentelemetry/trace/span.py

```diff
@@ -37,6 +42,13 @@ def set_attribute(self, key: str, value: types.AttributeValue) -> None:
         Sets a single Attribute with the key and value passed as arguments.
         """
 
+    @abc.abstractmethod
+    def get_attribute(self, key: str) -> types.AttributeValue:
+        """Get attribute.
+
+        Gets a single Attribute given a key and returns the attributes value
+        """
```

Comment:
Looking at the linked issue, was there a consensus on if this should be added to the API or just the SDK?

Reply:
I believe we needed to add it to the API. Such that code can be configured independent of the SDK

Reply:
Makes sense, but I'm not sure if the API is meant to follow the spec exactly? Can someone else comment on this?